Changelog.md

```diff
@@ -32,8 +32,8 @@
 
 - Exposed additional pins for I2S microphones. :bangbang: ESP32 boards only :bangbang:
 - Removed header 3 positions labeled AUDIO.
-- Added header 6 poitions labeled SOUND.
-- Added second row of headers for ESP32 boards connectivities.
+- Added header 6 position labeled SOUND.
```

Comment:
Header have a 6 positions

Reply:
True, they do indeed have six positions, but the text refers to a _single_ header, hence position as opposed to positions.